pyproject.toml

```diff
@@ -116,3 +116,5 @@ prerelease = true
 
 [tool.ruff]
 line-length = 120
+[tool.ruff.per-file-ignores]
+"__init__.py" = ["F401"]
```

Comment:
what's that?

Reply:
Look at this comment - https://github.com/AI21Labs/ai21-python/pull/84#discussion_r1538209780